script/openqa-clone-custom-git-refspec

```diff
@@ -21,22 +21,20 @@ Examples:
  openqa-clone-custom-git-refspec -n -c '--show-progress' https://github.com/coolgw/os-autoinst-distri-opensuse/tree/nfs https://openqa.opensuse.org/tests/835060 DESKTOP=textmode
  openqa-clone-custom-git-refspec https://github.com/foursixnine/os-autoinst-distri-opensuse/tree/oopsitsbrokenagain https://openqa.opensuse.org/tests/3128467 NEEDLES_DIR='https://github.com/foursixnine/os-autoinst-needles-opensuse.git#oopsitsbrokenagain'
 
-You need to set the environment variable GITHUB_TOKEN to use authenticated requests.
+To use authenticated requests, you need to either provide a GitHub token by
+adding 'GITHUB_TOKEN=...' to a configuration file at
+'~/.config/openqa/github-token.conf' or set the GITHUB_TOKEN environment variable.
```

Comment:
```suggestion
'github-token.conf' in your $OPENQA_CONFIG folder or set the GITHUB_TOKEN environment variable.
```

Mirroring the implementation, this shouldn't assume a specific folder. Might also be worth referring to the docs.

Reply:
https://open.qa/docs/#_customize_configuration_directory

Reply:
Right, it would be great it that feature would also support our usual way to customize the config directory.

Reply:
It does. It's just documented wrongly now 😁